src/components/leftCol/YourAccs.vue

```diff
@@ -71,6 +72,9 @@
       isLoadingAccounts(): boolean {
         return this.$store.state.lokapi.accountsLoading
       },
+      hasAccountsLoadingError(): boolean {
+        return this.$store.state.lokapi.accountsLoadingError
+      },
```

Comment:
I don't see why this would need to be put in the store as it concerns only this UI component. You are effectively adding variable in the store that are for only one component. You could have done a simple try/catch here and have the logic of this component kept close to the component.

If you had plans to use this in other components, this would have made sense to me.

No modification was required in the store with a lot of intelligence of individual components that makes it big and cluttered.

I noticed you've done what I suggest for `PendingAccounts` and `PendingCredits`.

Reply:
This is how @SeddikKadi did it, and I second his choice, because as the accounts list is created from the store with the `fetchAccounts` method , you cannot catch the error at the component level. It's not ideal but it's a tradeof, it's clean, and it works. If all the actions where to be tied from the components then what you suggest would be doable. It's not the case, and that's why it's done in PendingCredits and PendingAccounts on the component level, because this specific action is catchable from there.
But I'd be glad to know how you would do it, I may be missing something here.

Reply:
This one really is tricky. I'd be glad to receive your suggestion.